frontend/src/components/applyForm/validate.ts

```diff
@@ -106,17 +110,7 @@ export const validateUiSchema = (data: object) => {
   return validateJsonBySchema(data, UiJsonSchema);
 };
 
-const getKeysWithValues = (formData: FormData) => {
```

Comment:
This was removed for `shapeFormData` which gets the keys/values but also the correct field object shape.